### DISCUSSION TITLE
How to change the background color of a Dialog?

### DISCUSSION THREAD

**Discussion by marlonschlosshauer** [react-native-elements/react-native-elements#3593] [Q&A]
I'm trying to change the background color of the Dialog component. By default it's white, I want it to be lightblue. 
Here is what I tried:

```jsx
<Dialog
    style={{ backgroundColor: 'black' }}
    backdropStyle={{ backgroundColor: 'lightpink' }}
    overlayStyle={{ borderRadius: 16, borderWidth: 1, backgroundColor: 'black', borderColor: 'gold' }}
    isVisible={true}>
    <View style={{ backgroundColor: 'lightblue' }}>
	<Text style={{ fontSize: 20, textAlign: 'center' }}>
		Lorem ipsum dolor sit amet, consectetur adipiscing elit.
	</Text>
    </View>
</Dialog>
```

Screenshot of the above code:
<img src="https://user-images.githubusercontent.com/24259317/181495083-ffb2379d-4342-49e3-b221-2350c1ae0a3a.png" width="280"/>

`overlayStyle` applies `borderColor` (among others) but not `backgroundColor`. The `style` prop didn't change anything. Setting the background directly on the child leaves a gap. 

I couldn't find anything in the docs or in previous discussions/issues. A code snippet or link to docs would be awesome 🙌

**Comment by talhaakbartws** [comment]
Did anyone get answer?

**Reply by varendra007** [reply]
use `overlayStyle` to style your Dialog container.

```
<Dialog>    
    backdropStyle={{ backgroundColor: 'lightpink' }}
    overlayStyle={{ borderRadius: 16, borderWidth: 1, backgroundColor: 'lightblue', borderColor: 'gold' }}
    isVisible={true}>
	<Text style={{ fontSize: 20, textAlign: 'center' }}>
		Lorem ipsum dolor sit amet, consectetur adipiscing elit.
	</Text>
</Dialog>
```
It will work
